docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,489 @@
+        pulse.play(
+            pulse.library.Gaussian(duration=256, amp=0.2, sigma=50, name="custom"),
+            pulse.DriveChannel(0)
+        )
+    
+    circ.add_calibration("h", [0], h_q0)
+
+Call run on the circuit, and get counts as usual.
+
+.. jupyter-execute::
+
+    %time res = backend.run(circ).result()
+    
+    res.get_counts(0)
+
+4.2 Simulating circuits via gate definitions in the backend :class:`~qiskit.transpiler.Target`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively to the above work flow, add the above schedule as the pulse-level definition of the
+Hadamard gate on qubit :math:`0` to `backend.target`, which impacts how jobs are transpiled for the
+backend. See the :class:`~qiskit.transpiler.Target` class documentation for further information.
+
+.. jupyter-execute::
+
+    from qiskit.circuit.library import HGate
+    from qiskit.transpiler import InstructionProperties
+    
+    backend.target.add_instruction(HGate(), {(0,): InstructionProperties(calibration=h_q0)})
+
+Rebuild the same circuit, however this time we do not need to add the calibration for the Hadamard
+gate to the circuit object.
+
+.. jupyter-execute::
+
+    circ2 = QuantumCircuit(1, 1)
+    circ2.h(0)
+    circ2.measure([0], [0])
+    
+    %time result = backend.run(circ2).result()
+
+.. jupyter-execute::
+
+    result.get_counts(0)
+
+5. Simulating single-qubit calibration processes via Qiskit Experiments
+-----------------------------------------------------------------------
+
+Next, we perform rough calibrations for ``X`` and ``SX`` gates on both qubits modeled in the
+:class:`.DynamicsBackend`, following the single-qubit calibratino tutorial for Qiskit Experiments.
+
+5.1 Configure the :class:`~qiskit.transpiler.Target` to include single qubit instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable running of the single qubit experiments, we add the following to the target:
+
+- Qubit frequency properties (needed by experiments like 
+  :class:`~qiskit_experiments.library.calibration.rough_frequency.RoughFrequencyCal`).
+- ``X`` and ``SX`` gate instructions, which the transpiler needs to check are supported by the
+  backend. 
+- Add definitions of ``RZ`` gates as phase shifts.
+
+.. jupyter-execute::
+
+    from qiskit.circuit.library import XGate, SXGate, RZGate
+    from qiskit.circuit import Parameter
+    from qiskit.providers.backend import QubitProperties
+    
+    target = backend.target
+    
+    # qubit properties
+    target.qubit_properties = [QubitProperties(frequency=v0), QubitProperties(frequency=v1)]
+    
+    # add instructions
+    target.add_instruction(XGate())
+    target.add_instruction(SXGate())
+    
+    # Add RZ instruction as phase shift for drag cal
+    phi = Parameter("phi")
+    with pulse.build() as rz0:
+        pulse.shift_phase(phi, pulse.DriveChannel(0))
```

Comment:
Since this is a two qubit system, you also need to update control channel phase to run CR experiment, i.e. `ControlChannel(1)` for rz0.

Reply:
Ah thank you, good catch, I've added this.